pepper/apps/parsing.h

```diff
@@ -0,0 +1,71 @@
+#include <stdint.h>
+#include "dex_common.h"
+#include "macros.h"
+
+#define BITS_PER_ACCOUNT BITS_TO_REPRESENT(ACCOUNTS)
+#define BITS_PER_TOKEN BITS_TO_REPRESENT(TOKENS)
+#define BITS_PER_DECIMAL 100
+#define BITS_PER_ORDER (BITS_PER_ACCOUNT + (2 * BITS_PER_TOKEN) + (2 * BITS_PER_DECIMAL))
```

Comment:
Let's align here:

I proposed in the spread sheet:
accountLeafId (log2(Accounts)) +
fromId (log2(Token)) +
toId (log2(Token)) +
limit (price_bits) +
amount (bits per account balance B3
)

I think that a separation between bits for balance amount and bits for price makes quite some sense. 

The price should be more accurate in my opinion.